### ISSUE TITLE
More configuration options

### ISSUE THREAD

**Issue by jjrv** [phosphorjs/phosphor-dockpanel#14]
My use case is a large working area with panels around it. To avoid user confusion, I'd like to allow docking widgets only to the panels, not the working area. The main side panel will have at least one widget that cannot be closed, so it will remain available.

I'd like to have two features:
- Flag widget to disallow docking anything on top of it.
- Optional button in tabs to tear them off and float them as movable, resizable boxes over the working area (dragging them over the panels should invoke the current functionality to dock them back).

I could probably implement the first feature and submit a PR. Would that be helpful?

**Comment by vrmerlin** [comment]
I agree that these features would be really important, but am concerned that there was no response. Is this project still under active development? This docking panel looks like a great implementation, but I see very little in the way of documentation, or recent updates.

**Comment by sccolbert** [comment]
@vrmerlin Phosphor is still very much active. We've just consolidated everything back into a single repo, as maintaining thirty-one separate repos was becoming cumbersome. All work is now being done here:
https://github.com/phosphorjs/phosphor

Tear off panels is one of my 1.0 release criteria.

**Comment by vrmerlin** [comment]
ok, that's great news. I'll ask a follow-up question over on that project.
